Corrections and rewording

DraakjeYoblama opened the pull request:
Spelling and factual corrections, rewording some sections on Chapter 1-10. Only a few superficial edits on Chapter 11-13.

debber1 commented:
Hi! This is quite the PR, adding about 4 pages total and 600 lines of changed code. As you probably understand, this is a lot for us to handle, especially since we haven't taken this years course. We also have our own exams, so I don't think it is feasible for one of us to look at it in the coming days. Unfortunately, this means your PR will probably be put on hold for some time. 

Thanks for your efforts, and hopefully we'll get to it in the -not so distant- future. In the meantime (if you've got the time of course),  would you like to add a sort of change log, so that we have something to go by when reviewing your PR. 

Thanks again for the effort 
-debber

DraakjeYoblama commented:
I'll take a look at it after my exams are over, maybe edit the final chapters.

Change log probably will be shorter than expected, a lot of the changes are whitespace.

debber1 commented:
A general remark: 

I'm a fan of the restructuring of the code, as in the changes in indentation! However, in the future, please do this in a separate commit, as it makes the diff between commits essentially meaningless for the reviewer. 

There are a few "wasted" pages: 62, 65. they should be an ez fix by removing \newpage commands I would assume. 

As for the content changes, I am not going to check those properly nor do I think my colleagues will. We don't have access to the MOOC any more of course. 

Great changes overall. You made a lot of changes we have been meaning to make for a while but never got around to. 

Let me know if you want to look at the suggested changes yourself, or if I should just take care of them. Also, don't forget to add yourself and your link of choice to the contributors section in the readme and main tex file. (Or give us the information we need to do it ourselves.)

DraakjeYoblama commented:
- Removed the newpages
- Will do the indentation in a seperate commit next time
- Added myself to contributors
- Also copied Pieter Nagels from the readme contributors to the summary.tex file
